src/Microsoft.AspNetCore.SignalR.Core/HubClientsWrapperThing.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public class HubClientsWrapperThing : IHubCallerClients
+    {
+        private string _connectionId;
+        private IHubClients _hubClients;
+
+        public HubClientsWrapperThing(IHubClients hubClients, string ConnectionId)
+        {
+            _connectionId = ConnectionId;
+            _hubClients = hubClients;
+        }
+        public IHubClients HubClients => _hubClients;
+
+        public IClientProxy Caller => _hubClients.Client(_connectionId);
+
+        public IClientProxy Others => _hubClients.AllExcept(new List<string> { _connectionId });
```

Comment:
Initialize this list in the constructor so we don't create it every time you call `Others`.

Reply:
Also, `string[]` implements `IReadOnlyList<string>`, so just use an array.